src/security/ServerOptions.cc

```diff
@@ -474,6 +476,7 @@ Security::ServerOptions::updateContextEecdh(Security::ContextPointer &ctx)
         debugs(83, DBG_CRITICAL, "ERROR: EECDH is not available in this build." <<
                " Please link against OpenSSL>=0.9.8 and ensure OPENSSL_NO_ECDH is not set.");
 #endif
+        (void)ctx;
```

Comment:
This warning suppression missed its `#else` case.

Reply:
ok